armi/physics/neutronics/latticePhysics/latticePhysicsInterface.py

```diff
@@ -81,13 +81,6 @@ def __init__(self, r, cs):
 
         # Set to True by default, but should be disabled when perturbed cross sections are generated.
         self._updateBlockNeutronVelocities = True
-        # Geometry options available through the lattice physics interfaces
-        self._ZERO_DIMENSIONAL_GEOM = "0D"
-        self._ONE_DIMENSIONAL_GEOM = "1D"
-        self._TWO_DIMENSIONAL_GEOM = "2D"
-        self._SLAB_MODEL = " slab"
-        self._CYLINDER_MODEL = " cylinder"
-        self._HEX_MODEL = " hex"
```

Comment:
Are these removed because they are unused?

Reply:
I see they were used downstream but no longer are. Ok.

Reply:
They have been replaced by the `XSGeometryTypes` Enum. This could affect other users' downstream repos that use the `latticePhysicsInterface`, but it seems like good practice to force the usage of `XSGeometryTypes` throughout.

Reply:
@sombrereau @drewj-usnctech tagging for visibility